src/Microsoft.Health.Dicom.Core/Features/Update/IUpdateInstanceService.cs

```diff
@@ -0,0 +1,28 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Health.Dicom.Core.Models.Update;
+using Microsoft.Health.Operations;
+
+namespace Microsoft.Health.Dicom.Core.Features.Update;
+
+/// <summary>
+/// Provides functionality to queue the operation for updating the study attributes
+/// for a list of studyInstanceUids.
+/// </summary>
+public interface IUpdateInstanceService
```

Comment:
nit: Could we update to IUpdateOperationInstanceService

Since we cannot have the queuing and the update in the same service